geometry_msgs/Pose pose
---
float64[] jointPositions
bool success
